pkg/utils/kubernetes/health/health.go

```diff
@@ -19,6 +19,8 @@ import (
 	"net/http"
 	"time"
 
+	"github.com/gardener/gardener/pkg/utils/retry"
```

Comment:
Can you sort the imports, please?

Site note:
@gardener/gardener-maintainers We should probably add some extended import formatting tool to `make format`, so we can stop manually sorting the imports according to golang code style guidelines.
AFAIK @danielfoehrKn has a tool which could do the job: https://github.com/danielfoehrKn/importsort
Can we use that, WDYT?

Reply:
Hm, yeah, we should have a tool + verification for it. However, probably it makes sense to stick to already existing (and well-adopted) tools like `goimports` instead of creating/maintaining our own? I'm open to adapt our conventions so that `goimports` can be used out-of-the-box. WDYT?

Reply:
Sure, I would also prefer to stick to `goimports` for example. But AFAIK, it doesn't offer any possibility to properly sort imports as we do (e.g. third-party block last, all first-party/third-party imports in same block and so on), right?
Do you propose to accept any arbitrary sorting accepted by `goimports` now?

Reply:
Yes, that's why @danielfoehrKn created his own tool. I think `goimports` doesn't do it arbitrarily but adds the third-party imports before the repository-local imports. So, yes, I'm open to change this convention to get rid of the many "please sort the imports" review comments. :)

Reply:
Yeah, probably we should just agree on using `goimports` from now on and stop the "please sort the imports" review comments as long as it is accepted by `goimports`.

Reply:
Yep agree. Though I just automated the tool usage by adding it to my local pre-commit hook  ;(